servicemanager/servicemanager.go

```diff
@@ -209,7 +211,12 @@ func (sm *ServiceManager) doInstallService(newService ServiceInfo, imageURL stri
 		}
 	}
 
-	if err = sm.prepareServiceFS(newService.ImagePath, newService.GID); err != nil {
+	footFSDigest, err := sm.prepareServiceFS(newService.ImagePath, newService.GID)
+	if err != nil {
+		return aoserrors.Wrap(err)
+	}
+
+	if err = updateRootFSDigestInManifest(newService.ImagePath, footFSDigest); err != nil {
```

Comment:
Could be part of prepareServiceFS?

Reply:
from my point of view, manifest update should be saparate call